tests/Fixer/FunctionNotation/VoidReturnFixerTest.php

```diff
@@ -55,6 +55,20 @@ public function provideFixCases()
                  */
                 function foo($param) {}
             '],
+            ['<?php
+                /**
+                 * @return void|array
```

Comment:
`void` functions still return null:
```
php > $a = function(): void {};
php > var_dump($a());
NULL
```
I'm removing this though since it's a point of contention and I'm no longer worried about this use case due to the updated if statement.